### ISSUE TITLE
Illegal value when saving Map object

### ISSUE THREAD

**Issue by Laubeee** [sunpy/sunpy#3536]
I'm trying to convert a [RHESSI CLEAN image cube fits](https://hesperia.gsfc.nasa.gov/rhessi_extras/flare_images/2002/02/12/20020212_2130_2141/CLEAN/hsi_imagecube_clean_20020212_2130_6tx2e.fits) file into a `Map` object, and save it as new fits file. However I am getting errors `Illegal value: [6.0, 25.0]`
These two values come from `ENERGY_L` and `ENERGY_H`. However, what kind of value is expected there? I can't leave them away either, then it will complain upon creation of the `Map`.

### Steps to Reproduce
```python
from sunpy.io.fits import read
from sunpy.map import Map
f = read(path)
m = Map(f[0].data[0][0], f[0].header)  # extract first image into map
m.save("map.fits")
```

produces:
```---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
<ipython-input-106-f327d4b383fa> in <module>
      1 f = read(os.path.join(folder, files[2]))
      2 m = Map(f[0].data[0][0], f[0].header)  # extract first image into map
----> 3 m.save("map.fits")

~\Anaconda3\lib\site-packages\sunpy\map\mapbase.py in save(self, filepath, filetype, **kwargs)
    930         """
    931         io.write_file(filepath, self.data, self.meta, filetype=filetype,
--> 932                       **kwargs)
    933 
    934 # #### Image processing routines #### #

~\Anaconda3\lib\site-packages\sunpy\io\file_tools.py in write_file(fname, data, header, filetype, **kwargs)
    155         for extension, readername in _known_extensions.items():
    156             if fname.endswith(extension):
--> 157                 return _readers[readername].write(fname, data, header, **kwargs)
    158 
    159     else:

~\Anaconda3\lib\site-packages\sunpy\io\fits.py in write(fname, data, header, **kwargs)
    193 
    194         else:
--> 195             fits_header.append(fits.Card(k, v))
    196 
    197     if isinstance(key_comments, dict):

~\Anaconda3\lib\site-packages\astropy\io\fits\card.py in __init__(self, keyword, value, comment, **kwargs)
    191                 self.keyword = keyword
    192             if value is not None:
--> 193                 self.value = value
    194 
    195         if comment is not None:

~\Anaconda3\lib\site-packages\astropy\io\fits\card.py in value(self, value)
    319                            np.floating, np.integer, np.complexfloating,
    320                            np.bool_)):
--> 321             raise ValueError('Illegal value: {!r}.'.format(value))
    322 
    323         if isinstance(value, float) and (np.isnan(value) or np.isinf(value)):

ValueError: Illegal value: [6.0, 25.0].
```

### System Details
General
Time : Friday, 22. November 2019 04:31PM UT
System : Windows
Processor : Intel64 Family 6 Model 142 Stepping 9, GenuineIntel
Arch : 64bit
SunPy : 1.0.5
OS: Windows 10 10.0.18362 (Intel64 Family 6 Model 142 Stepping 9, GenuineIntel)

Required Libraries
Python: 3.7.2
NumPy: 1.16.4
SciPy: 1.3.0
matplotlib: 3.1.1
Astropy: 3.2.1
Pandas: 0.25.0
parfive: 1.0.0

**Comment by Cadair** [comment]
Hello @Laubeee I think this should be fixed by #3501 

If you fancy installing the master branch you can test it, if you don't want to do that you can wait a few hours and I hope to have 1.1rc1 out you could test.

**Comment by Laubeee** [comment]
worked with SunPy : 1.1.dev769+g56eef12c3
thanks!
